youtube_dl/options.py

```diff
@@ -298,6 +298,15 @@ def _scrub_eq(o):
         metavar='DATE', dest='dateafter', default=None,
         help='Download only videos uploaded on or after this date (i.e. inclusive)')
     selection.add_option(
+        '--date-playlist-order',
+        metavar='ORDER', dest='date_playlist_order', default='none',
+        type='choice', choices=['asc', 'desc', 'none'],
+        help='Specify whether a playlist is known to be listed in chronological order. "Asc", "desc" or "none". Descending order is most recent to least, and is useful for batch-downloading the new videos from a Youtube channel. Default is "none", and is like not using this option at all. Causes youtube-dl to automatically stop after encountering the first video date outside of the date range configured by --date, --datebefore, and/or --dateafter.')
```

Comment:
Should be cut down to coupe of sentences. Possible arguments' values should be written all lowercase without quotes. See `--fixup` for an example.